proof/crefine/RISCV64/IpcCancel_C.thy

```diff
@@ -7,7 +7,7 @@
  *)
 
 theory IpcCancel_C
-imports SyscallArgs_C
+imports SyscallArgs_C "HOL-Library.Divides"
```

Comment:
I forgot what this was for -- there was some lemma that was not available any more, but I think we might be able to eliminate the import again.